### ISSUE TITLE
Set language in league_prefs

### ISSUE THREAD

**Issue by Shteve0** [TheNAF/naflm#117]
Hi guys

Went to take a look at this tonight but I find I'm over-reaching my talents a little, I don't want to disrupt the css stuff in there that Craig's already done.

In short we're so close with the league prefs... if we could pull the language (default:en-EN), base TV and core theme from the leagueprefs menu we'd have no need for admin assistance in setting up leagues at all (and would be a step closer to the dream of eliminating settings_lid files entirely)

**Comment by rythos42** [comment]
I've avoided database changes, because I feel like the upgrade.php process isn't really visible and needs work before we do such things.

So I think I wouldn't do any work to try to put these values into the `leagueprefs` database table.

However, when I did Core Theme on the League Preferences UI, I just read and write to settings_lid, which is pretty safe of a change. I'd be ok doing Language and TV in this manner as well.

**Comment by Shteve0** [comment]
So is this a bug?

I went in to the dev site today and was able to change the base skin for DemoBowl to _greenwhite, hit save.  Went to the home page, it's correct (but not attractive, I should work more on these).  Then I went back to leagueprefs and changed to CHOP/Ausbowl and the skins say they're updated, but when I return to the league home page it's back to _greenwhite, even after clearing cache, and has reverted to _greenwhite in the leagueprefs menu.
